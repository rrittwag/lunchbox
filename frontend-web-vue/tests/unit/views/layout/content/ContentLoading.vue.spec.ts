import ContentLoading from '@/views/layout/content/ContentLoading.vue'
import { mount } from '@vue/test-utils'

describe('ContentLoading', () => {
  test('renders snapshot', () => {
    const wrapper = mount(ContentLoading)
    expect(wrapper.element).toMatchSnapshot()
  })
})
