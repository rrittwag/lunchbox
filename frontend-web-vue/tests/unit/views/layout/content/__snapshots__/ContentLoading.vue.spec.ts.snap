// Vitest Snapshot v1

exports[`ContentLoading > renders snapshot 1`] = `
<h1>
  Loading
</h1>
`;

exports[`ContentLoading renders snapshot 1`] = `
<h1>
  Loading
</h1>
`;
